ksqldb-functional-tests/src/test/resources/query-validation-tests/fk-multi-join.json

```diff
@@ -0,0 +1,294 @@
+{
```

Comment:
multi-joins is not a good name... Can we call it `fk-n-way-joins` instead?

I hope we can rename existing `mutli-joins.json`, too (after I figured out how to migrate generated history plans)